source/_integrations/bliss_automation.markdown

```diff
@@ -1,6 +1,6 @@
 ---
-title: Bliss Automation
-description: Connect and control your Bliss Automation devices using the Motion Blinds integration
+title: Bliss by Alta Automation
```

Comment:
This title taken from the integration manifest in our core codebase.
Was this automated there too?

Could you link the PR?